bsp/boards/openmote-b/radio_2d4ghz.c

```diff
@@ -71,7 +71,6 @@ void radio_2d4ghz_setFunctions(radio_functions_t* funcs){
     funcs->radio_rfOff_cb              = radio_2d4ghz_rfOff;
     funcs->radio_setFrequency_cb       = radio_2d4ghz_setFrequency;
     funcs->radio_change_modulation_cb  = radio_2d4ghz_change_modulation;
-    funcs->radio_change_size_cb        = radio_2d4ghz_change_size;
```

Comment:
if this callback function is not used, you need remove it from the openradio.h file as well.

Also the radio_2d4ghz_change_size function should be removed